adoc/admin-centralized-logging.adoc

```diff
@@ -0,0 +1,199 @@
+====
+
+Example RFC 3164
+----
+2019-05-30T09:11:21.968458+00:00 worker1 k8s.system/crio[12080] level=debug msg="Endpoint successfully created" containerID=caa46f14a68e766b877af01442e58731845bb45d8ce1f856553440a02c958b2f eventUUID=e2405f2a-82ba-11e9-9a06-fa163eebdfd6 subsys=cilium-cni
+----
+
+Example RFC 5424
+----
+<133>1 2019-05-30T08:28:38.784214+00:00 master0 k8s.pod/kube-system/kube-apiserver-master0/kube-apiserver - - [kube_meta namespace_id=1e030def-81db-11e9-a62b-fa163e1876c9 container_name=kube-apiserver creation_timestamp=2019-05-29T06:29:31Z host=master0 namespace_name=kube-system master_url=https://kubernetes.default.svc.cluster.local:443 pod_id=4aaf10f9-81db-11e9-a62b-fa163e1876c9 pod_name=kube-apiserver-master0] 2019-05-30T08:28:38.783780355+00:00 stderr F I0530 08:28:38.783710       1 log.go:172] http: TLS handshake error from 172.28.0.19:45888: tls: client offered only unsupported versions: [300]
+----
+
+
+== Installing Helm and Tiller
+
+In order to successfully use Centralized Logging, you first need to install `Helm` and `Tiller`.
+Helm is used to install the log agents and provide custom logging settings.
+
+- As of {productname} {productversion},
+Helm is part of the {productname} package repository, so to use Centralized Logging,
+you only need to run the following command from
+the location where you normally run `skuba` commands:
+
+[source,bash]
+----
+sudo zypper install helm
+----
+
+- As of {productname} {productversion},
+Tiller is not part of the {productname} package repository,
+so to install the Tiller container image run:
+
+[source,bash]
+----
+kubectl create serviceaccount --namespace kube-system tiller
+kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+
+helm init --tiller-image registry.suse.com/caasp/v4/helm-tiller:2.8.2 --service-account tiller
+----
+
+If the installation runs successfully there will be no output message.
+
+////
+Note: When Helm is included in v4, Tiller server will be automatically installed after CaaS Platform setup.
+So we probably  just need to mention that we use it and that it's installed automatically.
+////
+
+== Deployment
+
+After you have successfully installed it,
+use Helm CLI to install log agents on each node,
+and provide customized settings via specific command options.
+
+The only three mandatory parameters for a successful deployment of Centralized Logging
+are:
+
+* `*server.host*`, default value = rsyslog-server.default.svc.cluster.local
+* `*server.port*`, default value = 514
+* `*server.protocol*`, default value = TCP
+
+See <<Optional settings>> for the facultative parameters and their default values.
+
+- Running the following will create the minimal working setup:
+
+[source,bash]
+----
+helm repo add suse-charts https://kubernetes-charts.suse.com
+helm install stable/log-agent-rsyslog --name ${RELEASE_NAME} --set server.host=${SERVER_HOST} --set server.port=${SERVER_PORT}
```

Comment:
@nkoranova Just found small issue after merge into https://kubernetes-charts.suse.com

Could you please help change `helm install stable/log-agent-rsyslog` to `helm install suse-charts/log-agent-rsyslog`? Thanks.